website_and_docs/content/documentation/webdriver/waits.zh-cn.md

```diff
@@ -1,70 +1,47 @@
-but loaded JavaScript assets often result in changes to the site,
-and elements that need to be interacted with may not yet be on the page
-when the code is ready to execute the next Selenium command.
-
-Similarly, in a lot of single page applications, elements get dynamically
-added to a page or change visibility based on a click.
-An element must be both present and
-[displayed]({{< ref "elements/information/#is-displayed" >}}) on the page
-in order for Selenium to interact with it.
-
-Take this page for example: https://www.selenium.dev/selenium/web/dynamic.html
-When the "Add a box!" button is clicked, a "div" element that does not exist is created.
-When the "Reveal a new input" button is clicked, a hidden text field element is displayed.
-In both cases the transition takes a couple seconds.
-If the Selenium code is to click one of these buttons and interact with the resulting element,
-it will do so before that element is ready and fail.
-
-The first solution many people turn to is adding a sleep statement to
-pause the code execution for a set period of time.
-Because the code can't know exactly how long it needs to wait, this
-can fail when it doesn't sleep long enough. Alternately, if the value is set too high
-and a sleep statement is added in every place it is needed, the duration of
-the session can become prohibitive.
-
-Selenium provides two different mechanisms for synchronization that are better.
-
-
-## Implicit waits
-Selenium has a built-in way to automatically wait for elements called an _implicit wait_.
-An implicit wait value can be set either with the [timeouts]({{< ref "drivers/options#timeouts" >}})
-capability in the browser options, or with a driver method (as shown below).
-
-This is a global setting that applies to every element location call for the entire session.
-The default value is `0`, which means that if the element is not found, it will
-immediately return an error. If an implicit wait is set, the driver will wait for the 
-duration of the provided value before returning the error. Note that as soon as the 
-element is located, the driver will return the element reference and the code will continue executing, 
-so a larger implicit wait value won't necessarily increase the duration of the session.
-
-*Warning:*
-Do not mix implicit and explicit waits.
-Doing so can cause unpredictable wait times.
-For example, setting an implicit wait of 10 seconds
-and an explicit wait of 15 seconds
-could cause a timeout to occur after 20 seconds.
-
-Solving our example with an implicit wait looks like this:
+浏览器自动处理最大的挑战可能是确保网页在某个确定的、期望的状态下执行 Selenium 命令。
+这些过程通常处于 混乱竞争状态, 有时候网页先处理好，有时候 Selenium 命令先执行。
+这就是为什么需要稳定性测试。
 
+所有导航命令都需要一个准确的 `Ready状态` 值, 这取决于 [网页加载策略]({{< ref "drivers/options#pageloadstrategy" >}}) (一个默认的完成值)，之后`浏览器 Driver`将控制权交回给代码进行处理。
+这个 `Ready状态` 只关心在html 中定义的 css js 是否被加载，但 js 加载通常导致网页资源的变动，执行下一个 Selenium 命令时，网页元素在交互后可能还没有出现在网页中，。
```

Comment:
The assets are not only including css and js, it might be more generally treating as resources.
refer: https://www.selenium.dev/documentation/webdriver/drivers/options/#pageloadstrategy

Reply:
what should i do. replace "drivers/options#pageloadstrategy" to "https://www.selenium.dev/documentation/webdriver/drivers/options/#pageloadstrategy" ?

Reply:
Hi, Just mention that you translated the "assets" as "css js" , but actually, that's not meaning of them only...